android/app/src/main/java/app/organicmaps/MapFragment.java

```diff
@@ -210,6 +210,7 @@ private void reportUnsupported()
         .show();
   }
 
+  @SuppressWarnings("deprecated")
```

Comment:
Why is it suppressed? A clarifying comment is needed if it is a deliberate choice instead of fixing it instead.

Reply:
Because it's for old APIs, like a compat function.

Reply:
Sorry, I did not understand that. If we do not want/do not plan to fix this deprecation warning, there should be a reason for it. Can you please specify this reason in a comment near this line or near the corresponding API call, so anyone can immediately understand why suppression is added?